keras_preprocessing/sequence.py

```diff
@@ -23,7 +23,7 @@ def pad_sequences(sequences, maxlen=None, dtype='int32',
     or the length of the longest sequence otherwise.
 
     Sequences that are shorter than `num_timesteps`
-    are padded with `value` at the end.
+    are padded with `value` at the beginning.
```

Comment:
or the end if `truncating='post'`

Reply:
I will make this change to the PR today. Could you help me understand why the travis CI tests failed so I can fix that as well?

Reply:
Did you mean padding?